zowe/zowe-client-python-sdk
# fix: respect common env variables

**What It Does**


Fixes #346.
Added support for commonly used environmental variables, like `REQUESTS_CA_BUNDLE` and `CURL_CA_BUNDLE`

**How to Test**

1. Make sure you can run a sample command against an LPAR with valid certificates
    - `print((console.issue_command("D T"))['cmd-response'])`
2. Setup a proxy that will function as your custom Certificate Authority.
    - Steps: [Proxy configuration](https://phoenixnap.com/kb/setup-install-squid-proxy-server-ubuntu)
3. Specify the proxy as an environmental variable (either in your python script or in the command line)
    - `HTTPS_PROXY=https://myproxy.company.net:3129 python test.py`
    - Notice the `[SSL: CERTIFICATE_VERIFY_FAILED]` error
4. Specify either of the commonly used CA bundle variables with the proper certificate chain combination
    - `REQUESTS_CA_BUNDLE=/path/to/combo.crt`
    - `CURL_CA_BUNDLE=/path/to/combo.crt`
    - Note: The `combo.crt` refers to the complete certificate chain (proxyCA, intermediate/LPAR, global/root)
    - The command should work as expected 

**Review Checklist**
I certify that I have:


**Additional Comments**

The same can be achieved today using the `SSL_CERT_FILE` environmental variable
